static/docs/commands-reference/pull.md

```diff
@@ -1,53 +1,123 @@
 # pull
 
-Pulls data files to the working space.
+Pulls data files to the local workspace from a remote workspace.
```

Comment:
`from a remote storage` or just `remote`? I think in case of external outputs it is not always `local` (workspace can be distributed).

Reply:
Ah.. I should have written "remote DVC cache"?